packages/dx-react-grid/docs/reference/table-band-header.md

```diff
@@ -0,0 +1,68 @@
+# TableBandHeader Plugin Reference
+
+A plugin that renders the nested header cells.
+
+## User Reference
+
+### Dependencies
+
+- [Table](table.md)
+- [TableHeaderRow](table-header-row.md)
+- [TableEditColumn](table-edit-column.md) [Optional]
+- [TableSelection](table-selection.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+cellComponent | ComponentType&lt;[Table.CellProps](table.md#tablecellprops)&gt; | | A component that renders a parent cell.
```

Comment:
Where are the value and column properties? Is the column property really needed? Can value (title) be passed as a children to ease customization?

Reply:
You should add rowSpan to the Table.CellProps interface

Reply:
The cellComponent don't have rowSpan